CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We have switched to [CalVer](https://calver.org/)! Relay's version is always in
 
 **Internal**:
 
+- Add source (who emitted the outcome) to Outcome payload.([#604](https://github.com/getsentry/relay/pull/604))
```

Comment:
This is a super tiny nitpick: Please add a space before the parenthesis. 

Also, should we maybe collapse the changelog entries for #609 and #604 since they basically touch the same thing? Imho, the changelog doesn't have to be a commit log. See the entry for 584 and 591, for instance.

```suggestion
- Add source (who emitted the outcome) to Outcome payload. ([#604](https://github.com/getsentry/relay/pull/604))
```